### ISSUE TITLE
IE9 support

### ISSUE THREAD

**Issue by matthew-andrews** [Financial-Times/next-component-tools#14]
We support IE9… @aintgoin2goa seemed to suggest this doesn't work on IE9 currently…… woudl be worth checking……

**Comment by wheresrhys** [comment]
yep, need to wrap demo js in a js-setup.bootstrap() call or the polyfills won't be loaded

**Comment by wheresrhys** [comment]
Don't want to add hack upon hack to fix this until we've discussed where this module is going this afternoon. A temporary fix is to add this https://github.com/Financial-Times/next-js-setup/blob/master/templates/script-loader.html#L34 in a synchronous script tag at the top of the demo.
